tests/models/unit_cases_gas/source_sink/input/timeseries.csv

```diff
@@ -1,4 +1,4 @@
 DateTime,GasDemand_a2d8
-01-01-2019 00:00, 0.05
-01-01-2019 01:00, 0.1
-01-01-2019 02:00, 0.15
+01-01-2019 00:00, 0.045
```

Comment:
What is the reason for changing these values?

Reply:
Due to changes in the exact dimensions of the pipes, slightly smaller demands had to be chosen to ensure that we are still on the first, second, thrid line of the linearized lines for headloss in the tests.